Model/Config/Frontend/CronStatus.php

```diff
@@ -43,11 +43,13 @@ protected function _getElementHtml(AbstractElement $element)
         if ($item && $item->getId()) {
             if($item->getScheduledAt()){
                 $nextRun = new \DateTime($item->getScheduledAt());
-                $nextRunFormatted = $nextRun->modify('+1 Hour')->format('Y-m-d H:i:s');
+                $nextRunFormatted = $nextRun->modify('+1 Hour')->format('Y-m-d H:i:s T');
             }else{
                 $nextRunFormatted = 'Not Scheduled';
             }
-            $status = 'Last Run: ' . $item->getExecutedAt() . ' - Next Run: ' . $nextRunFormatted  . ' - Status: ' . $item->getStatus();
+            $lastRun = new \DateTime($item->getExecutedAt());
+            $lastRunFormatted = $lastRun->format('Y-m-d H:i:s T');
+            $status = 'Last Run: ' . $lastRunFormatted . ' - Next Run: ' . $nextRunFormatted  . ' - Status: ' . $item->getStatus();
```

Comment:
Might be nice to turn this into a translatable string
```suggestion
            $status = __(
                'Last Run: %1 - Next Run:  %2 - Status: %3',
                $lastRunFormatted,
                $nextRunFormatted,
                $item->getStatus()
            );
```